testing/s3minio/src/main/java/org/projectnessie/minio/MinioExtension.java

```diff
@@ -35,10 +40,20 @@
  * <p>Provides instances of {@link MinioAccess} via instance or static fields or parameters
  * annotated with {@link Minio}.
  */
-public class MinioExtension implements BeforeAllCallback, BeforeEachCallback, ParameterResolver {
+public class MinioExtension
+    implements BeforeAllCallback, BeforeEachCallback, ParameterResolver, ExecutionCondition {
   private static final ExtensionContext.Namespace NAMESPACE =
       ExtensionContext.Namespace.create(MinioExtension.class);
 
+  @Override
+  public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext context) {
+    if (OS.current() == OS.LINUX) {
+      return enabled("Running on Linux");
+    }
+    return disabled(
+        "Not running on Linux, because neither macOS nor Windows support wildcard localhost FQDNs.");
```

Comment:
Hmmm... I'm not sure I follow the meaning of this message :thinking:

Reply:
`foobar.localhost` cannot be resolved

Reply:
If we disable Minio tests on Mac+Win, how could we make sure our S3-related code still works there?

Reply:
(to clarify: I'm fine with this change as long as we fix the message, but I wonder if we actually need some other test env. for S3 on Mac/Win)

Reply:
They already do not work on those platforms, this change only let's CI pass on those platforms (see #5705).
The _mocked_ S3 impl is still exercised - Minio is not.

Reply:
Could we add a local host wildcards in a CI action?

Reply:
Too complicated and doesn't work - already tried.

Reply:
ok

Reply:
Could you just fix the "disabled" message? It still looks awkward :)

Reply:
It's a wildcard fully-qualified-domain-name?

Reply:
No, why `not running on Linux` when the OS is _not_ Linux?

Reply:
I'd say `Disabled on a non-Lixus OS, because....`

Reply:
I tend to interpret the disabled message as belonging to the test :) so in my mind  `the test is` `not running on Linux, becase...` looks odd :)

Reply:
Proposal?